doc/rbd/iscsi-target-cli-manual-install.rst

```diff
@@ -26,36 +44,68 @@ on each machine that will be a iSCSI gateway:
 -  python flask
 -  pyOpenSSL
 
-**Packages**
 
--  Linux Kernel
+2. Install Git
+==============
 
-   If not using a distro kernel that contains the required Ceph iSCSI patches,
-   then Linux kernel v4.16 or newer or the ceph-client ceph-iscsi-test
-   branch must be used. To get the branch run:
+In order to install all the packages needed to run iSCSI with Ceph, you need to download them directly from they repository by using Git.
+On CentOS/RHEL execute:
 
-   ::
+::
+
+   > sudo yum install git
+
+On Debian/Ubuntu execute:
+
+::
+
+   > sudo apt install git
+   
+To know more about Git and how it works please check https://git-scm.com
+
+
+3. Ensure to use a compatible kernel
+====================================
+
+Ensure to use a supported kernel that contains the required Ceph iSCSI patches:
+
+-  all Linux distribution with a kernel v4.16 or newer, or
+-  Red Hat Enterprise Linux or CentOS 7.5 or later (in these distributions ceph-iscsi support is backported in old kernel)
 
+If you are already using one of this alternative then you can go to next step.
+Instead if you NOT using any of those alternative then ceph-client ceph-iscsi-test
+branch must be used. To get the branch run:
+    
+    ::
+    
        > git clone https://github.com/ceph/ceph-client.git
        > git checkout ceph-iscsi-test
-
-   .. warning::
+    
+    .. warning::
        ceph-iscsi-test is not for production use. It should only be used
        for proof of concept setups and testing. The kernel is only updated
        with Ceph iSCSI patches. General security and bug fixes from upstream
        are not applied.
-
-   Check your distro's docs for specific instructions on how to build a
-   kernel. The only Ceph iSCSI specific requirements are the following
-   build options must be enabled:
-
-   ::
-
+    
+    Check your distro's docs for specific instructions on how to build a
+    kernel. The only Ceph iSCSI specific requirements are the following
+    build options must be enabled:
+    
+    ::
+    
        CONFIG_TARGET_CORE=m
        CONFIG_TCM_USER2=m
        CONFIG_ISCSI_TARGET=m
 
--  tcmu-runner
+
+4. Install ceph-iscsi-cli
+========================================================
+
+Finally we can start to fetch all the remaining packages directly from the their Git repos' and start the associated services.
```

Comment:
Nit: "Finally, the remaining tools can be fetched directly from their Git repositories and their associated services started."